fastify.d.ts

```diff
@@ -316,7 +316,7 @@ declare namespace fastify {
       req: NodeJS.ReadableStream,
       res: http.ServerResponse
     },
-    headers: object,
+    headers: Record<string,string>,
```

Comment:
```suggestion
    headers: Record<string, string>,
```
The linter is failing.